### PR TITLE
JAPI-72 Correctly use WsClient WsFS methods

### PR DESCRIPTION
**Pull request by rpastrana**
- Utilizes wsfsclient get dropzones

Signed-off-by: Pastrana, Rodrigo <rodrigo.pastrana@lexisnexis.com>

## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [x] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).

## Checklist:
- [x] I have created a corresponding JIRA ticket for this submission
- [x] My code follows the code style of this project.
  - [x] I have applied the Eclipse code-format template provided.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the HPCC Systems CONTRIBUTORS document (https://github.com/hpcc-systems/HPCC-Platform/wiki/Guide-for-contributors).
- [x] The change has been fully tested:
  - [ ] I have performed unit tests to cover my changes.
  - [ ] I have performed system test and covered possible regressions and side effects.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.

### CODE REVIEW

**Comment by rpastrana** [comment]
@Michael-Gardner please review.

**Comment by rpastrana** [comment]
@Michael-Gardner bump

File: rdf2hpcc/src/main/java/org/hpccsystems/rdf/rdf2hpcc/RDFHPCCWsClient.java [hunk 1]
```diff
@@ -821,41 +818,32 @@ private boolean setDropZoneInfo() throws Exception
...
+            if (dropZones.length > 0)
             {
-                DropZone[] dropZones = dropZoneFilesResponse.getDropZones();
-                if (dropZones.length > 0)
-                {
-                    targetDropzoneNetAddres = dropZones[0].getNetAddress();
-                    targetHPCCDropzonePath = dropZones[0].getPath();
-                    Utils.println(System.out, "Found dropzone net address: " + targetDropzoneNetAddres, false, verbosemode);
-                    Utils.println(System.out, "Found dropzone path: " + targetHPCCDropzonePath, false, verbosemode);
+                targetDropzoneNetAddres = dropZones[0].getNetAddress();
```
Comment (Michael-Gardner) [inline rdf2hpcc/src/main/java/org/hpccsystems/rdf/rdf2hpcc/RDFHPCCWsClient.java]: targetDropzoneNetAddres is missing an s on the end.  It's consistent throughout the file but I'd rather it be spelled correctly.

**Comment by rpastrana** [comment]
@Michael-Gardner misspelling addressed. Please merge. Should we updated the version number?

**Comment by Michael-Gardner** [comment]
No, I'll update version numbers when I push the release.

**Pull request merged**